File: Rock, Paper, Scissors/rockpaperscissors2.java
import java.util.Scanner;

public class rockpaperscissors2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        char player1 = scanner.next().charAt(0);
        char player2 = scanner.next().charAt(0);

        if (player1 == player2) {
            System.out.println("Draw");
        } else if ((player1 == 's' && player2 == 'p') || (player1 == 'r' && player2 == 's') || (player1 == 'p' && player2 == 'r')) {
            System.out.println("Player 1");
        } else {
            System.out.println("Player 2");
        }
    }
}

File: Rock, Paper, Scissors/rockpaperscissors2.c
#include <stdio.h>

int main() {
    char player1[9], player2[9];
    scanf("%s", player1);
    scanf("%s", player2);

    if (player1[0] == player2[0]) {
        printf("Draw");
    } else if ((player1[0] == 's' && player2[0] == 'p') || (player1[0] == 'r' && player2[0] == 's') || (player1[0] == 'p' && player2[0] == 'r')) {
        printf("Player 1"); 
    } else {
        printf("Player 2");
    }

    return 0;
}
